tests/test_asyncio.py

```diff
@@ -54,25 +53,6 @@ def test_all_tasks_no_loop() -> None:
         asyncio_backport.all_tasks()
 
 
-def test_all_tasks_threading() -> None:
```

Comment:
this test fails on all python versions anyway, so there's no point trying to backport the 3.7+ workarounds, because they don't work